src/Hl7.Fhir.Base/Serialization/engine/FhirSerializationEngineFactory.Legacy.cs

```diff
@@ -63,9 +63,31 @@ private enum Mode
         /// </summary>
         public static IFhirSerializationEngine FromParserSettings(ModelInspector inspector, ParserSettings settings) =>
             new ElementModelSerializationEngine(inspector,
-                BaseFhirParser.BuildXmlParsingSettings(settings),
-                BaseFhirParser.BuildJsonParserSettings(settings),
-                BaseFhirParser.BuildPocoBuilderSettings(settings));
+                buildXmlParsingSettings(settings),
+                buildJsonParserSettings(settings),
+                buildPocoBuilderSettings(settings));
+
+
+        private static PocoBuilderSettings buildPocoBuilderSettings(ParserSettings ps) =>
```

Comment:
Code moved old BaseParser, which has been deleted. This was the only place it is still used.